grails-app/controllers/org/broadinstitute/orsp/api/NewConsentGroupController.groovy

```diff
@@ -101,6 +85,7 @@ class NewConsentGroupController extends AuthenticatedController {
             if (consent != null) {
                 issueService.deleteIssue(consent.projectKey)
             }
+            consentCollectionLink?.delete(flush: true)
```

Comment:
nit: same feedback here, anything that hits the database should be in a service class method that can also be wrapped in a transaction.